### PR TITLE
docs for trying to be official docker images for Erlang OTP

### PR DESCRIPTION
**Pull request by c0b**
link with docker-library/official-images#1075

### CODE REVIEW

File: erlang/content.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> [wikipedia.org/wiki/Erlang_(programming_language)](https://en.wikipedia.org/wiki/Erlang_%28programming_language%29)
+
+%%LOGO%%
+
+# How to use this image
+
+## Create a `Dockerfile` in your Erlang app project
+
+```dockerfile
+FROM erlang:18-onbuild
```
Comment (yosifkit) [inline erlang/content.md]: You don't have an onbuild tag; probably just add the commands that would be in the onbuild. Something like this?

(warning, I am not an erlang user)
```dockerfile
FROM erlang:18
WORKDIR /usr/src/myapp/
COPY ./ /usr/src/myapp/
CMD ["escript", "myapp.erl"]
```

Comment (c0b) [inline erlang/content.md]: I'll remove these onbuild lines for now, will add the package manager `rebar` for the ONBUILD lines, it's like the pip for python
https://github.com/docker-library/python/blob/master/3.5/onbuild/Dockerfile

**Comment by tianon** [comment]
LGTM

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**